pages/onboarding/index.tsx

```diff
@@ -1,8 +1,60 @@
-import { Box, Button, Container, Typography } from '@mui/material';
+import { Box, Button, Container, TextField, Typography } from '@mui/material';
+import axios from 'axios';
+import jwt from 'jsonwebtoken';
 import { NextPage } from 'next';
 import Link from 'next/link';
+import { useRouter } from 'next/router';
+import React, { useState } from 'react';
+import Cookies from 'universal-cookie';
+
+import createAxiosInstance from 'client/lib/axios';
+import { UserProfile } from 'server/types/User';
+import { AUTH_COOKIE_NAME } from 'shared/constants';
+
+interface FormData {
+  bio: string;
+  twitter_username: string;
+  linkedin_url: string;
+  github_username: string;
+  website: string;
+}
+
+interface UseridData {
+  user: UserProfile;
+}
+
+const OnboardingPage: NextPage<UseridData> = ({ user }: UseridData) => {
```

Comment:
per your `getInitialProps`, `user` here could be `null`, so we should handle that case accordingly, otherwise the form won't work

Reply:
should we re-route the user to another page or set up an error?

Reply:
i would redirect to the home page, since we don't have a dedicated login page